v1/openapi/requests/jet-drop-byid.yaml

```diff
@@ -1,6 +1,6 @@
 get:
   summary: Jet drop by ID
-  description: Gets a jet drop by `jet_drop_id`.
+  description: Gets a `jet_drop` by `jet_drop_id`.
```

Comment:
So what do I get exactly?
I'd say: Gets the contents of a `jet_drop` given a `jet_drop_id` as a path (or query?) parameter.

Reply:
path